packages/web/docs/src/pages/self-hosting/get-started.mdx

```diff
@@ -41,13 +41,19 @@ in less than 15 minutes.
 
 First download the `docker-compose.community.yml` file from the
 [GitHub repository](https://github.com/kamilkisiela/graphql-hive/blob/main/docker-compose.community.yml)
-using `wget`.
+using `wget`. This's gonna download the latest version available.
+
+<Callout type="info">
+  Note: make sure you've docker installed to be able to continue with the setup process.
+</Callout>
 
 ```bash
 wget https://raw.githubusercontent.com/kamilkisiela/graphql-hive/main/docker-compose.community.yml
 ```
```

Comment:
You can also mention here: You can also download the file directly from GitHub, if you don't have `wget` installed.